package/scripts/auth.py

```diff
@@ -13,7 +13,6 @@
 # the License.
 #
 
-import sys
 import ambari_helpers as helpers
 from resource_management import *
```

Comment:
So... do I need to refactor the entire thing just to remove a couple unused imports, or can this be merged?